gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -167,7 +167,7 @@
             </ns0:Profile>
             <ns0:Group id="xccdf_mil.disa.stig_group_V-263651">
                 <ns0:Rule id="xccdf_mil.disa.stig_rule_SV-263651r1_rule" selected="true" severity="medium">
-                    <ns0:version>SRG-OS-263651-GPOS-263651</ns0:version>
+                    <ns0:version>SRG-OS-000690-GPOS-00140</ns0:version>
```

Comment:
I haven't validated the vast majority of these, but spot checking a few, they look correct.